### PR TITLE
fix golint errs

### PR DESCRIPTION
**Pull request by yanjing1104**

### CODE REVIEW

File: pkg/memtier/utils.go [hunk 1]
```diff
@@ -67,15 +67,18 @@ func sliceContainsInt(haystack []int, needle int) bool {
 	return false
 }
 
+/*
```
Comment (askervin) [inline pkg/memtier/utils.go:70]: Let's remove these functions if they are not used instead of commenting out.

**Review by askervin — COMMENTED** [review]
It looks very good, Jing! Thank you! Found only one thing to change.

**Comment by yanjing1104** [comment]
Most e2e test cases passed on ubuntu-22.04 with this PR (Skipped tracker-damon e2e test case).

jing@yanj1-desk1:~/go/src/github.com/intel/memtierd/test/e2e$ distro=ubuntu-22.04 reinstall_memtierd=1 k8scri=containerd ./run_tests.sh memtierd.test-suite/no-policy/
Tests summary:
PASS no-policy/n4c16/test00-routines
PASS no-policy/n4c16/test01-mover

jing@yanj1-desk1:~/go/src/github.com/intel/memtierd/test/e2e$ distro=ubuntu-22.04 reinstall_memtierd=1 k8scri=containerd ./run_tests.sh memtierd.test-suite/pidwatcher/
Tests summary:
PASS pidwatcher/n4c16/test00-pidlist
PASS pidwatcher/n4c16/test01-cgroups
PASS pidwatcher/n4c16/test02-filter-proc
PASS pidwatcher/n4c16/test03-filter-cgroups

jing@yanj1-desk1:~/go/src/github.com/intel/memtierd/test/e2e$ distro=ubuntu-22.04 reinstall_memtierd=1 k8scri=containerd ./run_tests.sh memtierd.test-suite/policy-age/
Tests summary:
PASS policy-age/n4c16/test00-swap-idlepage
PASS policy-age/n4c16/test01-move-softdirty

jing@yanj1-desk1:~/go/src/github.com/intel/memtierd/test/e2e$ distro=ubuntu-22.04 reinstall_memtierd=1 k8scri=containerd ./run_tests.sh memtierd.test-suite/policy-heat/
Tests summary:
PASS policy-heat/n4c16/test00-swap-idlepage
PASS policy-heat/n4c16/test01-swap-softdirty

jing@yanj1-desk1:~/go/src/github.com/intel/memtierd/test/e2e$ distro=ubuntu-22.04 reinstall_memtierd=1 k8scri=containerd ./run_tests.sh memtierd.test-suite/policy-ratio/
Tests summary:
PASS policy-ratio/n4c16/test00-swap
PASS policy-ratio/n4c16/test01-move

**Comment by yanjing1104** [comment]
With the improved CI workflow https://github.com/intel/memtierd/pull/35, this PR failed in the CodeQL check, mark it as WIP now.
Will rebase and fix this PR once https://github.com/intel/memtierd/pull/35 gets merged.

Check failure
Code scanning/ CodeQL
Incorrect conversion between integer typesHigh
Incorrect conversion of an unsigned 64-bit integer fromstrconv.ParseUintto a lower bit size type int64 without an upper bound check.

**Pull request merged**